version.go

```diff
@@ -1,7 +0,0 @@
-package skopeo
-
-// Version is a version of thils build.
-const Version = "0.1.12-dev"
```

Comment:
The `skopeo/signature` package from #25 embeds the `Version` into signatures (to allow future code to workaround past mistakes); that won't work when the `Version` is in the `main` package.

Basically, either the commands are in a subdirectory, or we need a `skopeo/version` subpackage importable by both the commands and `skopeo/signature`.

I don’t care all that much, I just wanted to highlight this tradeoff.